code/modules/atmospherics/FEA_gas_mixture.dm

```diff
@@ -4,19 +4,27 @@ What are the archived variables for?
 	This prevents race conditions that arise based on the order of tile processing.
 */
 
+/// Define a custom trace gas my making a subtype of this.
 /datum/gas
 	var/moles = 0
 #ifdef ATMOS_ARCHIVING
 	var/ARCHIVED(moles) = 0
 #endif
+	/// Amount of heat needed in joules to heat up 1 mole up 1 Kelvin.
 	var/specific_heat = 0
 
 /datum/gas/sleeping_agent
 	specific_heat = 40
+
 /datum/gas/oxygen_agent_b
 	specific_heat = 300
 
-
+/**
+ * The key of atmospherics.
+ * This datum here is how we represent gas mixtures.
+ * Temperature is in Kelvin and volume is in Litres.
+ * Todo cringe: write more about gas mixtures
```

Comment:
to people who don't know you're called cringe, this can be confusing. Try "cringe's todo:" or something. Because cringe is an adjective by itself and stuff. Yeah.

Reply:
its just temporary, will add more later